merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/model/InputType.java

```diff
@@ -170,7 +170,7 @@ T instantiate(Map<String, SerializedValue> arguments)
   List<ValidationNotice> getValidationFailures(T value);
 
   /** A named parameter to an {@link InputType} */
-  record Parameter(String name, ValueSchema schema) {}
+  record Parameter(String name, ValueSchema schema) { }
```

Comment:
This is an unneeded whitespace change and should be reverted:
```suggestion
  record Parameter(String name, ValueSchema schema) {}
```